modules/run/artifacts/opt/jboss/container/java/run/run-java.sh

```diff
@@ -93,15 +103,10 @@ load_env() {
 
   # Check also $JAVA_APP_DIR. Overrides other defaults
   # It's valid to set the app dir in the default script
-  if [ -z "${JAVA_APP_DIR}" ]; then
-    # XXX: is this correct?  This is defaulted above to /deployments.  Should we
-    # define a default to the old /opt/java-run?
-    JAVA_APP_DIR="${JBOSS_CONTAINER_JAVA_RUN_MODULE}"
-  else
-    if [ -f "${JAVA_APP_DIR}/${run_env_sh}" ]; then
+  if [ -f "${JAVA_APP_DIR}/${run_env_sh}" ]; then
```

Comment:
I think I might suggest we add some test coverage for this feature prior to merging this change. I'll sketch out how that look in another issue.